django_tiptap/widgets.py

```diff
@@ -29,4 +28,32 @@ def set_config(config: dict = None) -> dict:
     def get_context(self, *args, **kwargs) -> Dict[str, Any]:
         context = super().get_context(*args, **kwargs)
         context["widget"]["config"] = self.config
+
+        if (context["widget"]["config"].get("tooltips")):
+            return context
+        elif (context["widget"]["config"].get("lang")):
+            langs: list = ["EN", "DE"]
+            givenLang: str = context["widget"]["config"].get("lang")
+
+            if (givenLang.upper() in langs):
+                context["widget"]["config"]["tooltips"] = TIPTAP_DEFAULT_TOOLTIPS.copy()[givenLang.upper()]
+            else:
+                context["widget"]["config"]["tooltips"] = TIPTAP_DEFAULT_TOOLTIPS.copy()["EN"]
+                print('\n *** The language given to django_tiptap was not found, using english as default. *** \n')
+        else: 
+            context["widget"]["config"]["tooltips"] = TIPTAP_DEFAULT_TOOLTIPS.copy()["EN"]
+
+        if (context["widget"]["config"].get("translations")):
+            return context
+        elif (context["widget"]["config"].get("lang")):
+            langs: list = ["EN", "DE"]
+            givenLang: str = context["widget"]["config"].get("lang")
+
+            if (givenLang.upper() in langs):
+                context["widget"]["config"]["translations"] = TIPTAP_DEFAULT_TRANSLATIONS.copy()[givenLang.upper()]
+            else:
+                context["widget"]["config"]["translations"] = TIPTAP_DEFAULT_TRANSLATIONS.copy()["EN"]
+        else: 
+            context["widget"]["config"]["translations"] = TIPTAP_DEFAULT_TRANSLATIONS.copy()["EN"]
+
```

Comment:
What do you think about moving this into a function in `configs.py` in order to keep concerns seperated & nice and clean? :)

Reply:
yeah, that sounds cool, I'll move it there

Reply:
resolved